app/src/main/java/de/danoeh/antennapod/preferences/PreferenceController.java

```diff
@@ -641,7 +654,7 @@ private void setUpdateIntervalText() {
                 val = String.format(context.getString(R.string.pref_autoUpdateIntervallOrTime_at),
                         timeOfDayStr);
             } else {
-                val = context.getString(R.string.pref_smart_mark_as_played_disabled);
+                val = context.getString(R.string.pref_smart_mark_as_played_disabled);  // TODO: Is this a bug? Otherwise document why is this related to smart mark???
```

Comment:
Well, that variable should probably be shortened to ``disabled``. But that would mean that the string has to be re-translated for each and every language in transifex...